otel.go

```diff
@@ -0,0 +1,103 @@
+package logkeeper
+
+import (
+	"context"
+	"github.com/evergreen-ci/logkeeper/exporter"
+	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/semconv/v1.21.0"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"time"
+)
+
+const (
+	packageName = "github.com/evergreen-ci/logkeeper"
+)
+
+func (lk *logkeeper) initOtel(ctx context.Context) error {
+	if lk.opts.TraceCollectorEndpoint == "" {
```

Comment:
To me it would seem that if it's not configured we wouldn't want the spans to go out on stdout/grip either. Like if someone's trying to debug the application locally having so many spans in the output probably could get distracting.

Reply:
switched to default noop trace provider